<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerformSetWindow</class>
 <widget class="QMainWindow" name="PerformSetWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>20</width>
      <height>251</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>70</y>
      <width>20</width>
      <height>251</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="insertButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Вставити елемент</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Знайти елемент</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>40</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Видалити елемент</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="insertElement">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>250</width>
      <height>203</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Field name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Field Value</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="findElement">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>250</width>
      <height>203</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Field name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Field Value</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="deleteElement">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>100</y>
      <width>250</width>
      <height>203</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Field name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Field Value</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="foundIndicator">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>330</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="functionalLlabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Функціонал: Множина</string>
    </property>
   </widget>
   <widget class="QLabel" name="realisationLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Реалізація:</string>
    </property>
   </widget>
   <widget class="QLabel" name="contextLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Предметна область:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QComboBox" name="contentBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>445</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Сервери</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Дата і час</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>930</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
